src/main/webapp/script.js

```diff
@@ -131,5 +122,22 @@ function animate() {
   renderer.render(scene, camera);
 };
 
+/**
+ * This function rotates the data points about given axis using radians
+ */
+function rotateAxis(radians, axis){
+    for (let i = 0; i < rotationData.size; i++) {
+      if(axis == 'z') {
+        rotationData.get(i).origin.rotation.z = THREE.Math.degToRad(radians);
```

Comment:
`degToRad(radians)` doesn't look right. Either `radians` is mislabeled or you're double-converting.